priv/repo/migrations/20251126220802_add_last_boot_time_to_hosts.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Trento.Repo.Migrations.AddLastBootTimeToHosts do
```

Comment:
nit:
maybe we can rename the file and module to `AddLastBootTimeToHostReadModel` to follow the naming as we did before

Reply:
Done